client/src/ui/components/Button.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface ButtonProps extends React.HTMLProps<HTMLButtonElement> {
+  iconStart?: React.ReactNode;
+  iconEnd?: React.ReactNode;
+  color: 'red' | 'grey';
+  width: 'normal' | 'large';
+  isSubmit?: boolean;
+}
+
+export const Button = ({
+  children,
+  isSubmit,
+  color,
+  disabled,
+  width,
+  className,
+  iconEnd,
+  iconStart,
+  ...props
+}: ButtonProps) => (
+  <button
+    {...props}
+    type={isSubmit ? 'submit' : 'button'}
```

Comment:
It would be cleaner to just use the standard `type` prop and remove the isSubmit prop :)